### PR TITLE
feat: Updated the design of the category page

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

## Description

This PR will update the design of category page

**Pending**
Author section

Fixes https://linear.app/chatwoot/issue/CW-2621/update-the-design-of-the-category-page

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

**Screenshot**
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by muhsin-k — APPROVED** [review]

File: app/views/public/api/v1/portals/_uncategorized-block.html.erb [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <% portal.articles.published.where(category_id: nil).order(position: :asc).take(5).each do |article| %>
     <a
       class="text-slate-800 dark:text-slate-100 hover:underline leading-8"
-      href="/hc/<%= portal.slug %>/articles/<%= article.slug %><%= @theme.present? ? '?theme='+@theme : ''  %>"
+      href="/hc/<%= portal.slug %>/articles/<%= article.slug %><%= @theme.present? && @theme != 'system' ? '?theme='+@theme : ''  %>"
```
Comment (nithindavid) [inline app/views/public/api/v1/portals/_uncategorized-block.html.erb:14]: We could move this to a helper.

File: app/views/public/api/v1/portals/categories/_category-hero.html.erb [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
+    >
+      <%= I18n.t('public_portal.common.home') %>
+    </a>
+    <span class="w-4 h-4 [&>svg]:w-3 [&>svg]:h-3 flex items-center justify-center text-xs text-slate-500 dark:text-slate-300"><%= render partial: 'icons/chevron-right' %></span>
+    <span class="text-sm text-slate-800 dark:text-slate-100 font-semibold"><%= category.name %></span>
+  </div>
+  <div class="flex justify-start flex-col items-start w-full gap-2">
+    <div class="flex flex-col gap-2">
+      <% if category.icon.present? %>
+      <h1 class="text-4xl"><%= category.icon %></h1>
```
Comment (nithindavid) [inline app/views/public/api/v1/portals/categories/_category-hero.html.erb:15]: This doesn't need to be in h1 tag. We could use span on merge the two h1's

File: app/views/public/api/v1/portals/categories/_category-hero.html.erb [hunk 3]
```diff
@@ -1,17 +1,27 @@
...
+      <% end %>
+      <h1 class="text-3xl font-semibold leading-snug md:tracking-normal text-slate-900 dark:text-white">
```
Comment (nithindavid) [inline app/views/public/api/v1/portals/categories/_category-hero.html.erb:17]: The line height of title and body seems to be different from the figma design.

File: app/views/public/api/v1/portals/categories/_category-hero.html.erb [hunk 4]
```diff
@@ -1,17 +1,27 @@
...
+        <%= category.name %>
+      </h1>
+      <% if category.description.present? %>
+      <span class="font-medium text-slate-800 dark:text-slate-75 text-base"><%= category.description %></span>
+      <% end %>
     </div>
+    <%# >By Alexander and 3 others •%>
```
Comment (nithindavid) [inline app/views/public/api/v1/portals/categories/_category-hero.html.erb:24]: ✍️

File: app/views/public/api/v1/portals/categories/show.html.erb [hunk 5]
```diff
@@ -6,48 +6,37 @@
...
-          <p class="mb-2 hover:underline font-semibold"><%= article.title %></p>
-          <p class="text-sm">
-            <%= I18n.t('public_portal.common.last_updated_on', last_updated_on: article.updated_at.strftime("%b %d, %Y")) %>
-          </p>
+        <div class="flex flex-col gap-5">
+          <div class="flex flex-col gap-1">
+            <h3 class="text-lg text-slate-900 dark:text-slate-50 font-semibold <%= @is_plain_layout_enabled ? 'hover:underline' : 'group-hover:text-woot-600 dark:group-hover:text-woot-600' %>"><%= article.title %></h3>
+            <p class="text-base font-normal text-slate-500 dark:text-slate-200 line-clamp-1 break-all"><%= render_category_content(article.content) %></p>
+          </div>
+          <%# By Alexander •  %>
```
Comment (nithindavid) [inline app/views/public/api/v1/portals/categories/show.html.erb:35]: ✍️

**Review by nithindavid — CHANGES_REQUESTED** [review]
Please see the comments added

**Comment by scmmishra** [comment]
@iamsivin approved this, but let's merge the upstream PRs

**Pull request merged**